parsec/pubspec.yaml

```diff
@@ -3,15 +3,17 @@ description: Multi-platform `parsec` plugin for Flutter to calculate math equati
 version: 0.3.1
 repository: https://github.com/oxeanbits/parsec_flutter/tree/main/parsec
 
+publish_to: "none"
 environment:
-  sdk: ">=2.18.5 <3.0.0"
-  flutter: ">=2.5.0"
+  sdk: ">=3.3.0 <4.0.0"
+  flutter: ">=3.19.0"
 
 dependencies:
   flutter:
     sdk: flutter
   parsec_platform_interface: ^0.1.1
-  parsec_android: ^0.3.2
+  parsec_android: #^0.3.2
+    path: ../parsec_android
```

Comment:
This is only needed for testing, it will be removed in the next PR when the new version of parsec_android is published